layouts/community/ortho_3x10/wanleg/readme.md

```diff
@@ -83,4 +83,4 @@ Change `comPORT` to whatever port is used by the Arduino (e.g. `com11` in Window
 ## Using QMK DFU
 7. Once QMK DFU is burned to your ProMicro, you can then flash subsequent hex files with
 `make gherkin:<keymap>:dfu dfu=qmk`  
-The `dfu=qmk` conditional will set `BOOTLOADER = qmk-dfu` instead of `BOOTLOADER = caterina` 
+The `dfu=qmk` conditional will set `BOOTLOADER = qmk-dfu` instead of `BOOTLOADER = caterina`  
```

Comment:
You can also add `BOOTLOADER = qmk-dfu` in your rules.mk file, and it will "forever" be that way, so you don't need this option (since keymap overrides keyboard settings)